packages/dds/matrix/test/matrix.spec.ts

```diff
@@ -668,39 +668,6 @@ describe("Matrix", () => {
                     ["B", "A", 2, 3]
                 ]);
             });
```

Comment:
The below was me using the unit tests to repo/debug a stress failure.  It's redundant: we have a cleaner unit test above.